kie-server-parent/kie-server-services/kie-server-services-prometheus/src/main/java/org/kie/server/services/prometheus/PrometheusKieServerExtension.java

```diff
@@ -59,13 +72,62 @@ public boolean isInitialized() {
 
     @Override
     public boolean isActive() {
-        return !disabled;
+        return disabled == false;
```

Comment:
Nitpicking this change here - isn't this antipattern?

Reply:
fixed